### DISCUSSION TITLE
pipenv is not recognized as an internal or external command, operable program or batch file

### DISCUSSION THREAD

**Discussion by ramchaudhary2000** [Atri-Labs/atrilabs-engine#637] [Help: Installation & Start] [answered]
give the installing error 
<img width="465" alt="image" src="https://user-images.githubusercontent.com/73242657/209769565-c282ea7b-ed29-46da-941c-3559bcfffe35.png">

**Comment by darshitac11** [comment, marked as answer]
Please check the solution here: https://docs.atrilabs.com/troubleshoot/installation_error#pipenv-is-not-recognized-as-an-internal-or-external-command-operable-program-or-batch-file
